scripts/ci/run_perftest.sh

```diff
@@ -1,45 +1,47 @@
-#!/usr/bin/sh -ex
+#!/bin/bash -ex
```

Comment:
Seems like we use a lot of different shebangs in our scripts. Is this being changed because it wouldn't work with `sh` anymore and needs `bash`? I've seen us use `#!/usr/bin/env bash` quite a lot too.

Reply:
Yes I've used some bash-isms, especially `if [[` (two brackets) and `.` vs `source`.

Reply:
`/bin/bash` is more explicit that `/usr/bin/env bash` as it doesn't depend on the PATH before executing.